Dockerfile

```
@@ -1,163 +1,77 @@
-# Copy package files first for effective caching
-COPY renderer/package*.json ./
-RUN npm ci
-
-# Copy the rest of the renderer code and build frontend assets
-COPY renderer/ ./
-RUN npm run build && npm run postbuild
-
-# Set up chart-service
-WORKDIR /chart-service
+# Install build tools
+RUN apt-get update \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)